internal/action/command.go

```diff
@@ -198,7 +198,10 @@ func (h *BufPane) TabMoveCmd(args []string) {
 	idxTo = util.Clamp(idxTo, 0, len(Tabs.List)-1)
 
 	activeTab := Tabs.List[idxFrom]
-	Tabs.RemoveTab(activeTab.ID())
+	if len(activeTab.Panes) == 0 {
```

Comment:
I'd remove this check. If there is a tab without panes, it is a bug, right?

Reply:
True, I can remove it. 
I saw this check in `RemoveTab()` so I thought it was needed.

Reply:
In `RemoveTab()` it is probably not needed either... Also `RemoveTab()` looks quite weird overall: it checks specifically `p.Panes[0]`, not any other pane of the tab (not to mention that it is inconsistent with its documentation, which says that it checks the id of the tab itself).

Reply:
Yes, I was a bit confused as well but decided not to change it given that other places are using the pane id seems like.

Reply:
nit: "Removing redundant check on pane ID" in the commit message is gonna be confusing to the readers.